modal/_resolver.py

```diff
@@ -8,50 +8,80 @@
     Spinner = TypeVar("Spinner")
     Tree = TypeVar("Tree")
 
+from modal_utils.async_utils import synchronize_apis
 
-class Resolver:
+
+class StatusRow:
+    def __init__(self, progress):
```

Comment:
Perhaps type this as an `Optional[Tree]` - I was confused at first thinking it was the initial message

Reply:
👍